src/Connector.sol

```diff
@@ -9,7 +9,7 @@ import {MemberlistLike} from "./token/memberlist.sol";
 import {ConnectorMessages} from "src/Messages.sol";
 
 interface RouterLike {
```

Comment:
Why the like? Is this best practive in solidity?

Reply:
Yeah it seems to be a common practise suffix for Interfaces in Solidity

Reply:
Yeah common practice is either `*Like` or `I*` in Solidity